Requirements/src/consultant.feature

```diff
@@ -0,0 +1,54 @@
+Scenario: User navigates into consultant section
+Given	That a user is signed in
+When	 a signed in user comes to the consult section
+Then	should see list of consultants that are available
+And	should see input field to enter in Consultant name(if any)
+  
+Scenario: User navigates into consultant profile page
+Given	That a user selects a link provided from a consultant
+When	 a user navigates into consultant page
+Then	the user should see option to book appointment with consultant
+And	the user should see banner of recommended products from consultant
```

Comment:
what is a banner?

Reply:
A web banner, or banner ad, is a digital advertisement that appears on a web page. 
 The goal of a web banner is to drive traffic to the advertiser's website

![image](https://github.com/akshayvijaynair/poredocs/assets/27828994/96c13be6-3f89-4b06-9426-77b855c12716)